src/App.vue

```diff
@@ -24,10 +24,15 @@ const successCallback = (data, element) => {
   }
   if (data.hasOwnProperty("error")) {
     helper.error(t("ncdownloader", data.error));
-  } else if (data.hasOwnProperty("message")) {
-    helper.message(t("ncdownloader", data.message));
-  } else if (data.hasOwnProperty("file")) {
-    helper.message(t("ncdownloader", "Downloading" + " " + data.file));
+  } else {
+    if (data.hasOwnProperty("message")) {
+      helper.message(t("ncdownloader", data.message));
+    } else if (data.hasOwnProperty("file")) {
+      helper.message(t("ncdownloader", "Downloading" + " " + data.file));
+    }
+
+    document.getElementById("text-input-value").value = "";
+    document.querySelector(".active-downloads a").click();
```

Comment:
Not sure if it is the proper way to do this, though

Reply:
What do you want to achieve here? start polling immediately after loading the main page?

Reply:
Want to start polling right after the download was started. Without it "Active Downloads" count donesn't update immediately after starting the download

Reply:
for this, you just need to call  `helper.loop(helper.refresh, delay, ...["/apps/ncdownloader/status/active"])`

but I don't think we need this as you will eventually click on the tab if you'd like to check your download progress